### ISSUE TITLE
Correctly instrument SQL runtime in the `INFO` log level

### ISSUE THREAD

**Issue by benlovell** [rails/rails#27263]
Since 191facc857bb4fb52078fb544c6bc1613a81cc80 (particularly: https://github.com/rails/rails/commit/191facc857bb4fb52078fb544c6bc1613a81cc80#diff-eba6b59bacb9a1d88646145ed1ff4c86L34) the SQL runtime is not instrumented under the `INFO` log level. I guess this is a bug as I see no logical reason why this information is considered too verbose for the `INFO` log level. Indeed, this is still reported/emitted but as `0`.

Moving that line back above the `logger.debug?` guard corrects the issue.

I'm happy to submit a fix but wanted first to assess whether this was the expected behaviour.

### Steps to reproduce

https://gist.github.com/benlovell/3bd9093a6761d3b399517da517c8bdd8

### Expected behavior

The SQL runtime counter should be instrumented in all log levels.

### Actual behavior

The SQL runtime counter isn't instrumented correctly under the `INFO` log level.

### System configuration
**Rails version**: 5.0.0.1

**Ruby version**: 2.3.3

**Comment by maclover7** [comment]
cc @eileencodes

**Comment by rafaelfranca** [comment]
Yes, it is a bug. Instead of moving back the code can we change the conditional to:

```ruby
return unless logger.debug?

self.class.runtime += event.duration

return unless logger.info?
```

?

**Comment by benlovell** [comment]
Turns out this can be closed by #24963.
